incubator-tools/line_item_comparision/line_item_comparision.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        def get_line_items(doc):
</pre><p>A lot of these functions seem like they should be in the utilities instead of individual notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>